README.md

```diff
@@ -1,2 +1,51 @@
 # in-stock-bot
-A simple python telegram bot checking if a product is in stock
+* [![GitHub Stars](https://img.shields.io/github/stars/MayNiklas/in-stock-bot.svg)](stargazers)
+
+* [![GitHub Issues](https://img.shields.io/github/issues/MayNiklas/in-stock-bot.svg)](issues)
+
+* [![Build Status](https://travis-ci.org/joemccann/dillinger.svg?branch=master)](https://travis-ci.org/joemccann/dillinger)
+
+* [![Telegram Chat](https://img.shields.io/badge/Telegram-Join%20to%20us-blue.svg)](TelegramChannel)
+
+A small python Bot with Telegram notifications for checking availability of a product.
+
+![Chat Preview](./inStockBot.jpg)
+
+## Setup
+Clone this repo to your enviroment and change the `config.json` at first.
+In the `config.json` your must add your Telegram Api Key and one or multiple products.
+```json
+"product": [
+    {
+        "name": "Produkt Name",
+        "url": "https://product-url.domain/produkt/",
+        "value": "This Product Is Not Available"
+    },
+]
+```
+
+
+## Usage
+After you change the `config.json` File, build your Docker Container with 
+
+```sh 
+docker-compose up -d --build
```

Comment:
Strictly speaking, docker-compose up -d would be enough.
If the image hasn't been build yet -> it's build automatically.